src/index.html

```diff
@@ -5,8 +5,392 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list nav__list--position">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="hover">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <span class="content content--position">
```

Comment:
```suggestion
      <div class="content content--position">
```
more semantic to use div. span tag for text content